src/styles/utils/variables.scss

```
@@ -0,0 +1,15 @@
+$small-font-size: 10px;
+$active-color: #00acdc;
+$background-color: #fff;
+$textColor: #060b35;
+$grey-text-color: #616070;
+$card-border-radius: 5px;
+$button-border-radius: 5px;
+$cardWidth: 200px;
+$catalogGapX: 48px;
+$catalogGapY: 46px;
+$headerHeight: 60px;
```

It's not JS, so name your variables in one convention - using `-` to separate words. Fix this for all variables
```suggestion
$header-height: 60px;
```